scripts/run.sh

```diff
@@ -280,7 +233,9 @@ run_simulator() {
   --priority-fee=1
 }
 
-if [[ ${SKIP_NETWORK_RUNNER_START} == true ]]; then
+# We should only run scripts/parser/main.go if SKIP_NETWORK_RUNNER_START is set to 
+# false as this script starts up the network. 
+if [[ ${SKIP_NETWORK_RUNNER_START} != true ]]; then
```

Comment:
@gyuho could you confirm this is the right way to handle this? seems like this changes the previous behavior.

Reply:
Removed `parser/main.go` because the e2e_test.go starts up the network (or doesn't depending on the flag) there is no need for this file anymore -> `scripts/parser/main.go` as all it did was start up the network. Now the `SKIP_NETWORK_RUNNER_START`  flag works as expected.